**Project ID:** [plumID:19.075]({{ '/' | absolute_url }}eggs/19/075/)  
Stderr for source:  regtest/pycv/rt-f1/plumed.dat   
Download: [zipped raw stdout](plumed.dat.plumed.stdout.txt.zip) - [zipped raw stderr](plumed.dat.plumed.stderr.txt.zip) 
{% raw %}
<pre>
#! Only the first 1000 rows of the error file are shown below
#! To inspect the full error file, please download the zipped raw stderr file above
terminate called after throwing an instance of 'PLMD::Plumed::Exception'
what():
Action "PYTHONFUNCTION" is not known.
[pkrvm7jw40e0xgp:10510] *** Process received signal ***
[pkrvm7jw40e0xgp:10510] Signal: Aborted (6)
[pkrvm7jw40e0xgp:10510] Signal code:  (-6)
[pkrvm7jw40e0xgp:10510] [ 0] /lib/x86_64-linux-gnu/libc.so.6(+0x45330)[0x7f295fa45330]
[pkrvm7jw40e0xgp:10510] [ 1] /lib/x86_64-linux-gnu/libc.so.6(pthread_kill+0x11c)[0x7f295fa9eb2c]
[pkrvm7jw40e0xgp:10510] [ 2] /lib/x86_64-linux-gnu/libc.so.6(gsignal+0x1e)[0x7f295fa4527e]
[pkrvm7jw40e0xgp:10510] [ 3] /lib/x86_64-linux-gnu/libc.so.6(abort+0xdf)[0x7f295fa288ff]
[pkrvm7jw40e0xgp:10510] [ 4] /lib/x86_64-linux-gnu/libstdc++.so.6(+0xa5ff5)[0x7f295fea5ff5]
[pkrvm7jw40e0xgp:10510] [ 5] /lib/x86_64-linux-gnu/libstdc++.so.6(+0xbb0da)[0x7f295febb0da]
[pkrvm7jw40e0xgp:10510] [ 6] /lib/x86_64-linux-gnu/libstdc++.so.6(_ZSt10unexpectedv+0x0)[0x7f295fea5a55]
[pkrvm7jw40e0xgp:10510] [ 7] /lib/x86_64-linux-gnu/libstdc++.so.6(+0xa5a6f)[0x7f295fea5a6f]
[pkrvm7jw40e0xgp:10510] [ 8] plumed(+0x146dd)[0x5611226526dd]
[pkrvm7jw40e0xgp:10510] [ 9] /lib/x86_64-linux-gnu/libc.so.6(+0x2a1ca)[0x7f295fa2a1ca]
[pkrvm7jw40e0xgp:10510] [10] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0x8b)[0x7f295fa2a28b]
[pkrvm7jw40e0xgp:10510] [11] plumed(+0x15365)[0x561122653365]
[pkrvm7jw40e0xgp:10510] *** End of error message ***
</pre>
{% endraw %}
